sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamMicroBatchStream.scala

```diff
@@ -63,6 +63,7 @@ class RateStreamMicroBatchStream(
     val metadataLog =
       new HDFSMetadataLog[LongOffset](session.get, checkpointLocation) {
         override def serialize(metadata: LongOffset, out: OutputStream): Unit = {
+          // Writer should not close the given stream when overriding methods.
```

Comment:
Can you move this comment into the doc in the method as pointed out?

Reply:
> Can you move this comment into the doc in the method as pointed out?

Such as:
   // Writer should not close the given stream when overriding methods.
   override def serialize(metadata: LongOffset, out: OutputStream): Unit = { 

is ok?